code/modules/clothing/glasses/night.dm

```diff
@@ -70,6 +70,14 @@
 	name = "\improper M42 spotter sight"
 	desc = "A companion headset and night vision goggles system for USCM spotters. Allows highlighted imaging of surroundings. Click it to toggle."
 
+/obj/item/clothing/glasses/night/m42_night_goggles/spotter/can_use_active_effect(mob/living/carbon/human/user)
+	if(!(GLOB.character_traits[/datum/character_trait/skills/spotter] in user.traits))
+		to_chat(user, SPAN_WARNING("You have no idea how to use this!"))
```

Comment:
This message is unnecessary currently because the caller of this proc also sends a message. 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/b0fce6df-3fd5-4845-9d79-bc8f79e78848)